exchanges/bitstamp/bitstamp_wrapper.go

```diff
@@ -315,6 +318,37 @@ func (b *Bitstamp) UpdateTradablePairs(ctx context.Context, forceUpdate bool) er
 	return b.EnsureOnePairEnabled()
 }
 
+// UpdateOrderExecutionLimits sets exchange execution order limits for an asset type
+func (b *Bitstamp) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item) error {
+	if a != asset.Spot {
+		return common.ErrNotYetImplemented
+	}
+	symbols, err := b.GetTradingPairs(ctx)
+	if err != nil {
+		return err
+	}
+	limits := make([]order.MinMaxLevel, 0, len(symbols))
+	for x, info := range symbols {
+		if symbols[x].Trading != "Enabled" {
+			continue
+		}
+		pair, err := currency.NewPairFromString(symbols[x].Name)
+		if err != nil {
+			return err
+		}
+		limits = append(limits, order.MinMaxLevel{
+			Asset:                  a,
+			Pair:                   pair,
+			PriceStepIncrementSize: 1 / math.Pow10(info.CounterDecimals),
```

Comment:
If you can add in amount step that would be good as well:
```diff
diff --git a/exchanges/bitstamp/bitstamp_wrapper.go b/exchanges/bitstamp/bitstamp_wrapper.go
index 314ab47a7..96bd9e2b1 100644
--- a/exchanges/bitstamp/bitstamp_wrapper.go
+++ b/exchanges/bitstamp/bitstamp_wrapper.go
@@ -337,10 +337,11 @@ func (b *Bitstamp) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item)
                        return err
                }
                limits = append(limits, order.MinMaxLevel{
-                       Asset:                  a,
-                       Pair:                   pair,
-                       PriceStepIncrementSize: 1 / math.Pow10(info.CounterDecimals),
-                       MinimumBaseAmount:      info.MinimumOrder,
+                       Asset:                   a,
+                       Pair:                    pair,
+                       PriceStepIncrementSize:  1 / math.Pow10(info.CounterDecimals),
+                       AmountStepIncrementSize: 1 / math.Pow10(info.BaseDecimals),
+                       MinimumBaseAmount:       info.MinimumOrder,
                })
        }
        if err := b.LoadLimits(limits); err != nil {
```

Reply:
Step amount added